csatar-plugins/knowledgerepository/updates/builder_table_create_csatar_knowledgerepository_methodologies.php

```diff
@@ -0,0 +1,37 @@
+<?php namespace Csatar\KnowledgeRepository\Updates;
+
+use Schema;
+use October\Rain\Database\Updates\Migration;
+
+class BuilderTableCreateCsatarKnowledgerepositoryMethodologies extends Migration
+{
+    public function up()
+    {
+        Schema::create('csatar_knowledgerepository_methodologies', function($table)
+        {
+            $table->engine = 'InnoDB';
+            $table->increments('id')->unsigned();
+            $table->string('name')->nullable();
+            $table->text('description')->nullable();
+            $table->integer('timeframe_id')->nullable()->unsigned();
+            $table->integer('methodology_type_id')->nullable()->unsigned();
```

Comment:
Az ilyen esetekben, mint a 2 fenti sor, szerintem be kellene allitani az "index"-et. Legalabbis a csatar pluginon igy csinaltuk.